third_party/move/move-prover/bytecode/src/lib.rs

```diff
@@ -50,6 +46,9 @@ pub mod verification_analysis;
 pub mod verification_analysis_v2;
 pub mod well_formed_instrumentation;
 
+/// An error message used for cases where a compiled module is expected to be attached
+pub(crate) const COMPILED_MODULE_AVAILABLE: &str = "compiled module missing";
```

Comment:
Just a comment that this error message marks a significant portion of the refactoring and can be used to trace where a compiled module is expected in the code.

The name and the static string seems a bit out-of-sync at first sight, but when seeing how it is used, i.e., in `.expect(COMPILED_MODULE_AVAILABLE)`, the naming makes perfect sense.